python/cugraph_service/cugraph_service_server/cugraph_handler.py

```diff
@@ -157,14 +160,22 @@ def __del__(self):
 
     ###########################################################################
     # Environment management
-    @property
+    @cached_property
     def is_mg(self):
```

Comment:
I noticed there's a new API elsewhere called `is_multi_gpu()`.  Should this be renamed (or the new one renamed) so they match for consistency?

Reply:
`is_mg` is ambigious; I'll rename it.